tracer_manager/tracer_manager.F90

```diff
@@ -159,7 +159,7 @@ module tracer_manager_mod
 !> @{
 
 integer            :: num_tracer_fields = 0
-integer, parameter :: MAX_TRACER_FIELDS = 150
+integer, parameter :: MAX_TRACER_FIELDS = 250
```

Comment:
@GFDL-Eric can this be a namelist parameter?

Reply:
@thomas-robinson I don't believe we currently ingest a namelist for either tracer_manager or field_manager, although there are a set of tools in fm_utils.F90 that can "set" a namelist (fm_util_start_namelist), but it seems very ocean specific (hardcoded) and I'm not entirely sure what it's doing and/or if it's still even used. I don't know that we will have hardcoded limits like this after the rewrite, but that depends on how we implement things.